cosmetics-web/app/models/ingredient.rb

```diff
@@ -44,6 +44,7 @@ class Ingredient < ApplicationRecord
 
   validates :inci_name, presence: true, ingredient_name_format: { message: :invalid }
   validates :inci_name, uniqueness: { scope: :component_id }, if: :validate_inci_name_uniqueness?
+  validates :inci_name, length: { maximum: ResponsiblePersons::Notifications::IngredientConcentrationForm::NAME_LENGTH_LIMIT }
```

Comment:
Please change :raising_hand: 

This would invalidate around 500 already existing ingredients in production. When we introduced the form validation, we wanted to ensure that previously existing ingredients were still valid. That is why the length validation wasn´t here.



```suggestion
  validates :inci_name, length: { maximum: ResponsiblePersons::Notifications::IngredientConcentrationForm::NAME_LENGTH_LIMIT }
```
Another option is to restrict the validation only to be checked on creation.

Also, if we are sharing the constant, I would invert the dependency, so the `NAME_LENGTH_LIMIT` is defined in the model, not in the form.

Reply:
I will do `create` validation.